develop/plone/content/dynamic_views.rst

```diff
@@ -332,8 +331,8 @@ Migration script from default view to another
 ==============================================
 
 Below is a script snippet which allows you to change the default view
-for all folders to another type. You can execute the script through
-the :term:`ZMI` as a Python Script.
+for all folders to another type.
+You can execute the script through the Management Interface as a Python Script.
```

Comment:
"Script" should not be capitalized (unless you're referring to a "Script (Python)")